Date: Mon, 13 Nov 2023 20:05:34 +0100
Subject: [PATCH] fix: updated nora-w1 5.1 patch

---
 components/esp_wifi/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/esp_wifi/CMakeLists.txt b/components/esp_wifi/CMakeLists.txt
index bcd442eeb8..7647ff7ace 100644
--- a/components/esp_wifi/CMakeLists.txt
+++ b/components/esp_wifi/CMakeLists.txt
@@ -40,7 +40,7 @@ idf_component_register(SRCS "${srcs}"
                     INCLUDE_DIRS "include" "wifi_apps/include"
                     REQUIRES esp_event esp_phy esp_netif
                     PRIV_REQUIRES driver esptool_py esp_pm esp_timer nvs_flash
-                                  wpa_supplicant hal lwip esp_coex ${extra_priv_requires}
+                                  wpa_supplicant hal lwip esp_coex ubx_phy_cal ${extra_priv_requires}
                     LDFRAGMENTS "${ldfragments}")
 
 if(CONFIG_ESP_WIFI_ENABLED)
-- 
2.39.1.windows.1

