diff --git a/components/esp_phy/src/phy_init.c b/components/esp_phy/src/phy_init.c
index 27d6bbbe26..878def242b 100644
--- a/components/esp_phy/src/phy_init.c
+++ b/components/esp_phy/src/phy_init.c
@@ -477,11 +477,11 @@ void esp_phy_release_init_data(const esp_phy_init_data_t* init_data)
 
 // phy_init_data.h will declare static 'phy_init_data' variable initialized with default init data
 
-const esp_phy_init_data_t* esp_phy_get_init_data(void)
-{
-    ESP_LOGD(TAG, "loading PHY init data from application binary");
-    return &phy_init_data;
-}
+// const esp_phy_init_data_t* esp_phy_get_init_data(void)
+// {
+//     ESP_LOGD(TAG, "loading PHY init data from application binary");
+//     return &phy_init_data;
+// }
 
 void esp_phy_release_init_data(const esp_phy_init_data_t* init_data)
 {
diff --git a/components/esp_wifi/CMakeLists.txt b/components/esp_wifi/CMakeLists.txt
index 961bcccac7..7ce707be86 100644
--- a/components/esp_wifi/CMakeLists.txt
+++ b/components/esp_wifi/CMakeLists.txt
@@ -31,7 +31,7 @@ idf_component_register(SRCS "${srcs}"
                     INCLUDE_DIRS "include"
                     REQUIRES esp_event esp_phy
                     PRIV_REQUIRES driver esptool_py esp_netif esp_pm esp_timer nvs_flash
-                                  wpa_supplicant hal ${extra_priv_requires}
+                                  wpa_supplicant hal ubx_phy_cal ${extra_priv_requires}
                     LDFRAGMENTS "${ldfragments}")
 
 if(CONFIG_ESP32_WIFI_ENABLED)
